### DISCUSSION TITLE
Icon on Switch? (IconSwitch component)

### DISCUSSION THREAD

**Discussion by Hopp3r** [chakra-ui/chakra-ui#3432] [Ideas] [answered]
I'm not sure if this is already implemented, I didn't see a way to do it in the docs. 
Is there a way to include an icon on a switch? I'm thinking light/dark mode for example.

**Comment by mattwells19** [comment, marked as answer]
There very well may be a better way, but some special CSS seems to do the trick: https://codesandbox.io/s/zealous-flower-8b60y
